### ISSUE TITLE
Breadcrumb storybook should not include 'Playground' stories

### ISSUE THREAD

**Issue by behowell** [microsoft/fluentui#29242]
### Current 

Currently, the default breadcrumb story is a playground that allows modifying props through radio buttons. 

https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--default

<img width="514" alt="image" src="https://github.com/microsoft/fluentui/assets/48106640/334a310d-1feb-4235-b9b1-c3dd7f8cc9dc">

### Expected

A few guidelines for writing stories:
* The default story should be the simplest/default usage of the control.
* Avoid extra label text within the example code of the story (e.g. headers that describe each example). Splitting out the examples into separate stories should avoid the need for these labels.
* The stories shouldn't include playground controls (instead: include a separate story the different variants).
* Avoid stories that combine multiple types of variants in the same story (e.g. don't include separator variants in the **Size** story).
* When possible, avoid adding helper functions to the story (like `renderItem`) unless it's part of the recommended way to use it. In this case, the `BreadcrumbItem` elements should be written out longhand to show how it's used.

For example, the Default story could be something like this:
```jsx
export const Default = () => (
  <Breadcrumb aria-label="Example Breadcrumb">
    <BreadcrumbItem>Item 1</BreadcrumbItem>
    <BreadcrumbDivider />
    <BreadcrumbItem>Item 2</BreadcrumbItem>
    <BreadcrumbDivider />
    <BreadcrumbItem>Item 3</BreadcrumbItem>
    <BreadcrumbDivider />
    <BreadcrumbItem current>Item 4</BreadcrumbItem>
  </Breadcrumb>
);
```

And the Size story could look like:
```jsx
export const Size = () => (
  <>
    <Breadcrumb aria-label="Small Breadcrumb" size="small">
      <BreadcrumbItem>Small 1</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem>Small 2</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem current>Small 3</BreadcrumbItem>
    </Breadcrumb>
    <Breadcrumb aria-label="Medium Breadcrumb" size="medium">
      <BreadcrumbItem>Medium 1</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem>Medium 2</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem current>Medium 3</BreadcrumbItem>
    </Breadcrumb>
    <Breadcrumb aria-label="Large Breadcrumb" size="large">
      <BreadcrumbItem>Large 1</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem>Large 2</BreadcrumbItem>
      <BreadcrumbDivider />
      <BreadcrumbItem current>Large 3</BreadcrumbItem>
    </Breadcrumb>
  </>
);
```

A possible list of stories might be:
* Default
* Size
* Appearance: Subtle 
  * No need to include `transparent` since that's the default.
* Divider: Slash
  * No need to include `chevron` since that's the default.
* Buttons 
  * E.g. Similar to the Default one but with BreadcrumbButtons
* Focus Mode: Arrow
  * No need to include `tab` since that's the default.
* Overflow
* Tooltips

**Comment by ValentinaKozlova** [comment]
In case with Breadcrumb helper functions are recommended because otherwise the component will look too verbose. Using it without helper functions can be dome only for non-interactive items and when there's not many of them

**Comment by ValentinaKozlova** [comment]
@behowell , should I create a separate story for the size variant? Because BreacrumbButton is different from BreadcrumbItem in `small` size. That's how I currently added it:
![image](https://github.com/microsoft/fluentui/assets/11574680/4c26d729-f1ac-4619-8401-16fff0dd3bde)

**Comment by behowell** [comment]
@ValentinaKozlova 
> In case with Breadcrumb helper functions are recommended because otherwise the component will look too verbose. Using it without helper functions can be dome only for non-interactive items and when there's not many of them

In that case, it might be worth reconsidering the default API to make it less verbose? For example, it would make more sense to me if you could use `BreadcrumbButton` without a surrounding `BreadcrumbItem`. Or otherwise, then it might be worth exporting the helper functions so each user doesn't need to copy them.



> should I create a separate story for the size variant? Because BreacrumbButton is different from BreadcrumbItem in `small` size. 

I'd recommend having the `Size` story show the three sizes of BreadcrumbItem (small, medium, large). And in the description of the story, mention that BreadcrumbButton doesn't support small. For example:

"Breadcrumb's `size` can be `"small"`, `"medium"` (default), or `"large"`.  Note: when using BreadcrumbButton, the `"small"` size is not supported."

**Comment by ValentinaKozlova** [comment]
> @ValentinaKozlova
> 
> > In case with Breadcrumb helper functions are recommended because otherwise the component will look too verbose. Using it without helper functions can be dome only for non-interactive items and when there's not many of them
> 
> In that case, it might be worth reconsidering the default API to make it less verbose? For example, it would make more sense to me if you could use `BreadcrumbButton` without a surrounding `BreadcrumbItem`. Or otherwise, then it might be worth exporting the helper functions so each user doesn't need to copy them.
> 
> > should I create a separate story for the size variant? Because BreacrumbButton is different from BreadcrumbItem in `small` size.
> 
> I'd recommend having the `Size` story show the three sizes of BreadcrumbItem (small, medium, large). And in the description of the story, mention that BreadcrumbButton doesn't support small. For example:
> 
> "Breadcrumb's `size` can be `"small"`, `"medium"` (default), or `"large"`. Note: when using BreadcrumbButton, the `"small"` size is not supported."

That's the thing, we agreed on that API on the very beginning (spec review + tech sync). Also, the recommendation is anyway to use helper functions because people still will use `map` to iterate in items

**Comment by ValentinaKozlova** [comment]
> @ValentinaKozlova
> 
> > In case with Breadcrumb helper functions are recommended because otherwise the component will look too verbose. Using it without helper functions can be dome only for non-interactive items and when there's not many of them
> 
> In that case, it might be worth reconsidering the default API to make it less verbose? For example, it would make more sense to me if you could use `BreadcrumbButton` without a surrounding `BreadcrumbItem`. Or otherwise, then it might be worth exporting the helper functions so each user doesn't need to copy them.
> 
> > should I create a separate story for the size variant? Because BreacrumbButton is different from BreadcrumbItem in `small` size.
> 
> I'd recommend having the `Size` story show the three sizes of BreadcrumbItem (small, medium, large). And in the description of the story, mention that BreadcrumbButton doesn't support small. For example:
> 
> "Breadcrumb's `size` can be `"small"`, `"medium"` (default), or `"large"`. Note: when using BreadcrumbButton, the `"small"` size is not supported."

BreadcrumbButton support `small` it's just have different spacing. It was a specific request from the design team

**Comment by ValentinaKozlova** [comment]
Fixed in #29302
